backend/src/tasks/lightning/node-sync.service.ts

```diff
@@ -106,7 +108,22 @@ class NodeSyncService {
 
     try {
       // @ts-ignore
-      const [channels]: [ILightningApi.Channel[]] = await DB.query(`SELECT channels.id FROM channels WHERE channels.status = 1 AND ((SELECT COUNT(*) FROM nodes WHERE nodes.public_key = channels.node1_public_key) = 0 OR (SELECT COUNT(*) FROM nodes WHERE nodes.public_key = channels.node2_public_key) = 0)`);
+      const [channels]: [ILightningApi.Channel[]] = await DB.query(`
```

Comment:
Note: This is just a formatting update. The query is still the same but we can at least read it now

Reply:
I approve of this subtle trolling